jobbergate-cli/jobbergate_cli/subapps/job_scripts/tools.py

```diff
@@ -127,3 +129,23 @@ def create_job_script(
     )
 
     return job_script_result
+
+
+def download_job_script_files(
```

Comment:
This function would be better named `save_job_script_files`

Reply:
solved in [802a6e0](https://github.com/omnivector-solutions/jobbergate/pull/198/commits/802a6e0dbdd5c472b04832387af5b93e73949eb9)